modules/serverless-supported-configs.adoc

```diff
@@ -0,0 +1,4 @@
+[id="serverless-supported-configs_{context}"]
```

Comment:
This file is missing the metadata that says which assemblies it's used in.

Reply:
Oops, added thanks.